### PR TITLE
add desc for tiflash system tables

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12629
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-tiflash-tables.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+title: TIFLASH_TABLES
+summary: 了解 information_schema 表 `TIFLASH_TABLES`。
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-tables.md]:
```suggestion
summary: Learn the `TIFLASH_TABLES` information_schema table.
```

File: information-schema/information-schema-tiflash-tables.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+- `TABLE`: The name of the table in TiFlash.
+- `TIDB_DATABASE`: The name of the database that the table belongs to in TiDB.
+- `TIDB_TABLE`: The name of the table in TiDB.
+- `TABLE_ID`: The internal ID of the table, which is unique within a TiDB cluster.
+- `IS_TOMBSTONE`: Indicates whether the table can be recycled. `1` indicates that the table can be recycled, and `0` indicates that the table is normal.
+- `SEGMENT_COUNT`: The number of segments in the table. A segment is a data management unit in TiFlash.
+- `TOTAL_ROWS`: The total number of rows in the table.
+- `TOTAL_SIZE`: The total size of the table (in bytes).
+- `TOTAL_DELETE_RANGES`: The total number of Delete Ranges in the table.
+- `DELTA_RATE_ROWS`: The ratio of total rows of the table in the Delta layer to the total rows of that table.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-tables.md:93]:
```suggestion
- `DELTA_RATE_ROWS`: The ratio of the total rows of the table in the Delta layer to the total rows of that table.
```

File: information-schema/information-schema-tiflash-tables.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+---
+title: TIFLASH_TABLES
+summary: 了解 information_schema 表 `TIFLASH_TABLES`。
+---
+
+# TIFLASH_TABLES
+
+> **Warning:**
+>
+> Do not use this table in production environments, as the contents of the table are unstable, and subject to change in new releases of TiDB, without prior notice.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-tables.md:10]:
```suggestion
> Do not use this table in production environments, as the fields of the table are unstable, and subject to change in new releases of TiDB, without prior notice.
```

File: information-schema/information-schema-tiflash-segments.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+---
+title: TIFLASH_SEGMENTS
+summary: Learn the `TIFLASH_SEGMENTS` information_schema table.
+---
+
+# TIFLASH_SEGMENTS
+
+> **Warning:**
+>
+> Do not use this table in production environments, as the contents of the table are unstable, and subject to change in new releases of TiDB, without prior notice.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-segments.md:10]:
```suggestion
> Do not use this table in production environments, as the fields of the table are unstable, and subject to change in new releases of TiDB, without prior notice.
```

File: information-schema/information-schema-tiflash-segments.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+```
+
+Fields in the `TIFLASH_SEGMENTS` table are described as follows:
+
+- `DATABASE`: Database name in TiFlash. The segment belongs to a table in this database.
+- `TABLE`: Table name in TiFlash. The segment belongs to this table.
+- `TIDB_DATABASE`: Database name in TiDB. The segment belongs to a table in this database.
+- `TIDB_TABLE`: Table name in TiDB. The segment belongs to this table.
+- `TABLE_ID`: The internal ID of the table to which the segment belongs. This ID is unique within a TiDB cluster.
+- `IS_TOMBSTONE`: Indicates whether the table the segment belongs to can be recycled. `1` indicates that the table can be recycled. `0` indicates that the table is normal.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-segments.md:67]:
```suggestion
- `IS_TOMBSTONE`: Indicates whether the table the segment belongs to can be recycled. `1` indicates that the table can be recycled. `0` indicates that the table is in a normal state.
```

File: information-schema/information-schema-tiflash-segments.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+- `SEGMENT_ID`: Segment ID, which is unique within a table.
+- `RANGE`: The range of data that the segment contains.
+- `EPOCH`: The updated version of the segment. The version number of each segment is monotonically increasing.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-segments.md:70]:
```suggestion
- `EPOCH`: The updated version of the segment. The version number of each segment increases monotonically.
```

File: information-schema/information-schema-tiflash-segments.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+- `STABLE_PAGE_ID`: The disk storage ID of the data in the Stable layer.
+- `STABLE_ROWS`: The total number of rows in the Stable layer.
+- `STABLE_SIZE`: The total size of the data in the Stable layer (in bytes).
+- `STABLE_DMFILES`: The number of DMFiles in the Stable layer.
+- `STABLE_DMFILES_ID_0`: The disk storage ID of the first DMFile in the Stable layer.
+- `STABLE_DMFILES_ROWS`: The total number of rows in the DMFile in the Stable layer.
+- `STABLE_DMFILES_SIZE`: The total size of the data in the DMFile in the Stable layer (in bytes).
+- `STABLE_DMFILES_SIZE_ON_DISK`: The disk space occupied by the DMFile in the Stable layer (in bytes).
+- `STABLE_DMFILES_PACKS`: The number of Packs in the DMFile in the Stable layer.
+- `TIFLASH_INSTANCE`: TiFlash instance address.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-segments.md:94]:
```suggestion
- `TIFLASH_INSTANCE`: The address of the TiFlash instance.
```

File: information-schema/information-schema-tiflash-tables.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+```
+
+Fields in the `TIFLASH_TABLES` table are described as follows:
+
+- `DATABASE`: The name of the database that the table belongs to in TiFlash.
+- `TABLE`: The name of the table in TiFlash.
+- `TIDB_DATABASE`: The name of the database that the table belongs to in TiDB.
+- `TIDB_TABLE`: The name of the table in TiDB.
+- `TABLE_ID`: The internal ID of the table, which is unique within a TiDB cluster.
+- `IS_TOMBSTONE`: Indicates whether the table can be recycled. `1` indicates that the table can be recycled, and `0` indicates that the table is normal.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-tables.md:88]:
```suggestion
- `IS_TOMBSTONE`: Indicates whether the table can be recycled. `1` indicates that the table can be recycled, and `0` indicates that the table is in a normal state.
```

File: information-schema/information-schema-tiflash-tables.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+- `STORAGE_DELTA_NUM_SNAPSHOTS`: The number of snapshots in the Delta layer.
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_LIFETIME`: The duration of the earliest snapshot in the Delta layer (in seconds).
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_THREAD_ID`: The thread ID of the earliest snapshot in the Delta layer.
+- `STORAGE_DELTA_OLDEST_SNAPSHOT_TRACING_ID`: The tracing ID of the earliest snapshot in the Delta layer.
+- `STORAGE_META_NUM_SNAPSHOTS`: The number of snapshots in the meta information.
+- `STORAGE_META_OLDEST_SNAPSHOT_LIFETIME`: The duration of the earliest snapshot in the meta information (in seconds).
+- `STORAGE_META_OLDEST_SNAPSHOT_THREAD_ID`: The thread ID of the earliest snapshot in the meta information.
+- `STORAGE_META_OLDEST_SNAPSHOT_TRACING_ID`: The tracing ID of the earliest snapshot in the meta information.
+- `BACKGROUND_TASKS_LENGTH`: The length of the task queue in the background.
+- `TIFLASH_INSTANCE`: TiFlash instance address.
```
Comment (Oreoxmt) [inline information-schema/information-schema-tiflash-tables.md:136]:
```suggestion
- `TIFLASH_INSTANCE`: The address of the TiFlash instance.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: information-schema/information-schema-tiflash-tables.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+- `DELTA_RATE_ROWS`: The ratio of total rows of the table in the Delta layer to the total rows of that table.
+- `DELTA_RATE_SEGMENTS`: The proportion of segments that contain non-empty Delta layers in the table.
+- `DELTA_PLACED_RATE`: The proportion of rows that have completed index construction of the table in the Delta layer.
+- `DELTA_CACHE_SIZE`: The size of the cache of the table Delta layer (in bytes).
+- `DELTA_CACHE_RATE`: The proportion of cache data of the table Delta layer.
+- `DELTA_CACHE_WASTED_RATE`: The proportion of invalid cache data of the table Delta layer.
+- `DELTA_INDEX_SIZE`: The size of the memory occupied by indexes in the Delta layer (in bytes).
+- `AVG_SEGMENT_ROWS`: The average number of rows in all segments of the table.
+- `AVG_SEGMENT_SIZE`: The average size of all segments of the table (in bytes).
+- `DELTA_COUNT`: The number of segments of the table in the Delta layer.
```
Comment (lidezhu) [inline information-schema/information-schema-tiflash-tables.md:102]:
```suggestion
- `DELTA_COUNT`: The number of segments which contain a non-empty Delta layer in the table.
```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by lidezhu — APPROVED** [review]
LGTM

**Pull request merged**